pkg/api/handlers/compat/images_build.go

```diff
@@ -80,6 +80,7 @@ func BuildImage(w http.ResponseWriter, r *http.Request) {
 		Target      string   `schema:"target"`
 		Outputs     string   `schema:"outputs"`
 		Registry    string   `schema:"registry"`
+		IIDFile     string   `schema:"iidfile"`
```

Comment:
Why are we passing this through the API? This seems like it ought to be done by the frontend?

Reply:
(Or are we supposed to be creating the ID file on the server? That seems strange...)

Reply:
I would also expect this to be handled entirely by the client rather than the server.

Reply:
it cannot, as implemented, be handled by the client.  buildah does the writing of this to file.  it is hardwired.

Reply:
Doesn't buildah hand back to podman build the ID Can't it send the ID back to the client side so the id can be written to a file?

Reply:
Is this now having the client write out the IIDfile?

Reply:
Not yet.

I have a hard time imaging scenarios where the iidfile could be useful on the server. I am usually using such files for systemd services or scripts, but all of them would run on the client.
```bash
$ podman-remote build --iidfile=/tmp/foo ...
$ podman-remote run `cat /tmp/foo`
```
... this would work only if the service and client are on the same machine.

Reply:
If this does not work on my MAC with the server running on a different machine, then this is not fixed.